packages/flame/lib/src/game/camera.dart

```diff
@@ -142,8 +143,33 @@ class Camera {
   /// add any non-smooth movement.
   Rect? worldBounds;
 
+  /// If set, the camera will zoom by this ratio. This be greater than 1 (in)
+  /// or smaller (out) but should be always greater than zero.
```

Comment:
```suggestion
  /// or smaller (out) but should always be greater than zero.
```

Reply:
This sentence is a bit confusing over all, less than 1 should be zoom out right?

Reply:
yeah I forgot a `can` as well